// @expo-google-fonts/inter/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const Inter_100Thin = require('./100Thin/Inter_100Thin.ttf');
export const Inter_200ExtraLight = require('./200ExtraLight/Inter_200ExtraLight.ttf');
export const Inter_300Light = require('./300Light/Inter_300Light.ttf');
export const Inter_400Regular = require('./400Regular/Inter_400Regular.ttf');
export const Inter_500Medium = require('./500Medium/Inter_500Medium.ttf');
export const Inter_600SemiBold = require('./600SemiBold/Inter_600SemiBold.ttf');
export const Inter_700Bold = require('./700Bold/Inter_700Bold.ttf');
export const Inter_800ExtraBold = require('./800ExtraBold/Inter_800ExtraBold.ttf');
export const Inter_900Black = require('./900Black/Inter_900Black.ttf');
export const Inter_100Thin_Italic = require('./100Thin_Italic/Inter_100Thin_Italic.ttf');
export const Inter_200ExtraLight_Italic = require('./200ExtraLight_Italic/Inter_200ExtraLight_Italic.ttf');
export const Inter_300Light_Italic = require('./300Light_Italic/Inter_300Light_Italic.ttf');
export const Inter_400Regular_Italic = require('./400Regular_Italic/Inter_400Regular_Italic.ttf');
export const Inter_500Medium_Italic = require('./500Medium_Italic/Inter_500Medium_Italic.ttf');
export const Inter_600SemiBold_Italic = require('./600SemiBold_Italic/Inter_600SemiBold_Italic.ttf');
export const Inter_700Bold_Italic = require('./700Bold_Italic/Inter_700Bold_Italic.ttf');
export const Inter_800ExtraBold_Italic = require('./800ExtraBold_Italic/Inter_800ExtraBold_Italic.ttf');
export const Inter_900Black_Italic = require('./900Black_Italic/Inter_900Black_Italic.ttf');
