// @lucide/lab/dist/esm/lucide-lab.js
/**
 * @license @lucide/lab v0.1.2 - ISC
 *
 * This source code is licensed under the ISC license.
 * See the LICENSE file in the root directory of this source tree.
 */

export { default as ampersandSquare } from './icons/ampersand-square.js';
export { default as appleCore } from './icons/apple-core.js';
export { default as arrowsUpDownSquare } from './icons/arrows-up-down-square.js';
export { default as astronautHelmet } from './icons/astronaut-helmet.js';
export { default as atSignCircle } from './icons/at-sign-circle.js';
export { default as atSignSquare } from './icons/at-sign-square.js';
export { default as avocado } from './icons/avocado.js';
export { default as babyPacifier } from './icons/baby-pacifier.js';
export { default as bacon } from './icons/bacon.js';
export { default as bagHand } from './icons/bag-hand.js';
export { default as barbecue } from './icons/barbecue.js';
export { default as barberPole } from './icons/barber-pole.js';
export { default as barn } from './icons/barn.js';
export { default as baseball } from './icons/baseball.js';
export { default as baselineSquare } from './icons/baseline-square.js';
export { default as basketball } from './icons/basketball.js';
export { default as batBall } from './icons/bat-ball.js';
export { default as bathBubble } from './icons/bath-bubble.js';
export { default as beachBall } from './icons/beach-ball.js';
export { default as bearFace } from './icons/bear-face.js';
export { default as bedBunk } from './icons/bed-bunk.js';
export { default as beeHive } from './icons/bee-hive.js';
export { default as bee } from './icons/bee.js';
export { default as beetleScarab } from './icons/beetle-scarab.js';
export { default as bellConciergeDot } from './icons/bell-concierge-dot.js';
export { default as bellConciergeOff } from './icons/bell-concierge-off.js';
export { default as bellConcierge, default as conciergeBell } from './icons/bell-concierge.js';
export { default as belt } from './icons/belt.js';
export { default as boldSquare } from './icons/bold-square.js';
export { default as bottleBaby } from './icons/bottle-baby.js';
export { default as bottleChampagne } from './icons/bottle-champagne.js';
export { default as bottleDispenser } from './icons/bottle-dispenser.js';
export { default as bottlePerfume } from './icons/bottle-perfume.js';
export { default as bottlePlastic } from './icons/bottle-plastic.js';
export { default as bottleSpray } from './icons/bottle-spray.js';
export { default as bottleToothbrushComb } from './icons/bottle-toothbrush-comb.js';
export { default as bowlChopsticks } from './icons/bowl-chopsticks.js';
export { default as bowlOverflow } from './icons/bowl-overflow.js';
export { default as bowling } from './icons/bowling.js';
export { default as braSports } from './icons/bra-sports.js';
export { default as briefcasePlus } from './icons/briefcase-plus.js';
export { default as broom } from './icons/broom.js';
export { default as bucket } from './icons/bucket.js';
export { default as bullHead } from './icons/bull-head.js';
export { default as burger } from './icons/burger.js';
export { default as butterfly } from './icons/butterfly.js';
export { default as cabin } from './icons/cabin.js';
export { default as cabinetFiling } from './icons/cabinet-filing.js';
export { default as cactus } from './icons/cactus.js';
export { default as candleHolderLit } from './icons/candle-holder-lit.js';
export { default as candleHolder } from './icons/candle-holder.js';
export { default as candleTealightLit } from './icons/candle-tealight-lit.js';
export { default as candleTealight } from './icons/candle-tealight.js';
export { default as candlestickBigLit } from './icons/candlestick-big-lit.js';
export { default as candlestickBig } from './icons/candlestick-big.js';
export { default as candlestickLit } from './icons/candlestick-lit.js';
export { default as candlestick } from './icons/candlestick.js';
export { default as cardCredit } from './icons/card-credit.js';
export { default as cardSd } from './icons/card-sd.js';
export { default as cardSim } from './icons/card-sim.js';
export { default as cartonOff } from './icons/carton-off.js';
export { default as carton } from './icons/carton.js';
export { default as caseCamel } from './icons/case-camel.js';
export { default as caseKebab } from './icons/case-kebab.js';
export { default as caseSnakeUpper } from './icons/case-snake-upper.js';
export { default as caseSnake } from './icons/case-snake.js';
export { default as catBig } from './icons/cat-big.js';
export { default as cauldron } from './icons/cauldron.js';
export { default as centCircle } from './icons/cent-circle.js';
export { default as centSquare } from './icons/cent-square.js';
export { default as cent } from './icons/cent.js';
export { default as chairsTableParasol } from './icons/chairs-table-parasol.js';
export { default as chairsTablePlatter } from './icons/chairs-table-platter.js';
export { default as chameleon } from './icons/chameleon.js';
export { default as cheese } from './icons/cheese.js';
export { default as chest } from './icons/chest.js';
export { default as chevronsLeftRightSquare, default as squareCode } from './icons/chevrons-left-right-square.js';
export { default as chevronsUpDownSquare } from './icons/chevrons-up-down-square.js';
export { default as cloth } from './icons/cloth.js';
export { default as coatHanger } from './icons/coat-hanger.js';
export { default as cocktail } from './icons/cocktail.js';
export { default as coconut } from './icons/coconut.js';
export { default as coffeeBean } from './icons/coffee-bean.js';
export { default as coffeemaker } from './icons/coffeemaker.js';
export { default as coinsExchange } from './icons/coins-exchange.js';
export { default as coinsStack } from './icons/coins-stack.js';
export { default as copyCode } from './icons/copy-code.js';
export { default as copyDown } from './icons/copy-down.js';
export { default as copyFilePath } from './icons/copy-file-path.js';
export { default as copyImage } from './icons/copy-image.js';
export { default as copyText } from './icons/copy-text.js';
export { default as copyType } from './icons/copy-type.js';
export { default as cowHead } from './icons/cow-head.js';
export { default as cowUdderDroplets } from './icons/cow-udder-droplets.js';
export { default as crab } from './icons/crab.js';
export { default as cricketBall } from './icons/cricket-ball.js';
export { default as cricketWicket } from './icons/cricket-wicket.js';
export { default as crossSquare } from './icons/cross-square.js';
export { default as crosshair2Dot } from './icons/crosshair-2-dot.js';
export { default as crosshair2 } from './icons/crosshair-2.js';
export { default as crosshairPlusDot } from './icons/crosshair-plus-dot.js';
export { default as crosshairPlus } from './icons/crosshair-plus.js';
export { default as crosshairSquare } from './icons/crosshair-square.js';
export { default as cupSaucer } from './icons/cup-saucer.js';
export { default as cupToGo } from './icons/cup-to-go.js';
export { default as currencySquare } from './icons/currency-square.js';
export { default as deskLamp } from './icons/desk-lamp.js';
export { default as diaper } from './icons/diaper.js';
export { default as dishwasher } from './icons/dishwasher.js';
export { default as circleDollarSign, default as dollarSignCircle } from './icons/dollar-sign-circle.js';
export { default as dollarSignSquare } from './icons/dollar-sign-square.js';
export { default as doorbellIntercom } from './icons/doorbell-intercom.js';
export { default as dress } from './icons/dress.js';
export { default as eggCup } from './icons/egg-cup.js';
export { default as elephantFace } from './icons/elephant-face.js';
export { default as elephant } from './icons/elephant.js';
export { default as escalatorArrowDownLeft } from './icons/escalator-arrow-down-left.js';
export { default as escalatorArrowUpRight } from './icons/escalator-arrow-up-right.js';
export { default as euroCircle } from './icons/euro-circle.js';
export { default as euroSquare } from './icons/euro-square.js';
export { default as faceAlien } from './icons/face-alien.js';
export { default as fanHandheld } from './icons/fan-handheld.js';
export { default as farm } from './icons/farm.js';
export { default as faucet } from './icons/faucet.js';
export { default as featherPlus } from './icons/feather-plus.js';
export { default as featherSquare } from './icons/feather-square.js';
export { default as featherText } from './icons/feather-text.js';
export { default as flippers } from './icons/flippers.js';
export { default as floorPlan } from './icons/floor-plan.js';
export { default as floppyDisk2 } from './icons/floppy-disk-2.js';
export { default as floppyDiskRear } from './icons/floppy-disk-rear.js';
export { default as floppyDisk, default as save } from './icons/floppy-disk.js';
export { default as floppyDisks2 } from './icons/floppy-disks-2.js';
export { default as floppyDisksRear } from './icons/floppy-disks-rear.js';
export { default as floppyDisks, default as saveAll } from './icons/floppy-disks.js';
export { default as flowerLotus } from './icons/flower-lotus.js';
export { default as flowerPot } from './icons/flower-pot.js';
export { default as flowerRoseSingle } from './icons/flower-rose-single.js';
export { default as flowerRose } from './icons/flower-rose.js';
export { default as flower2, default as flowerStem } from './icons/flower-stem.js';
export { default as flowerTulip } from './icons/flower-tulip.js';
export { default as footballGoal } from './icons/football-goal.js';
export { default as footballHelmet } from './icons/football-helmet.js';
export { default as football } from './icons/football.js';
export { default as forkKnifeCrossed, default as utensilsCrossed } from './icons/fork-knife-crossed.js';
export { default as forkKnife, default as utensils } from './icons/fork-knife.js';
export { default as foxFaceTail } from './icons/fox-face-tail.js';
export { default as frogFace } from './icons/frog-face.js';
export { default as fruit } from './icons/fruit.js';
export { default as garlic } from './icons/garlic.js';
export { default as gearboxSquare } from './icons/gearbox-square.js';
export { default as gearbox } from './icons/gearbox.js';
export { default as gemRing } from './icons/gem-ring.js';
export { default as glassesSquare } from './icons/glasses-square.js';
export { default as glassesSun } from './icons/glasses-sun.js';
export { default as goalNet } from './icons/goal-net.js';
export { default as gobletCrack } from './icons/goblet-crack.js';
export { default as goblet } from './icons/goblet.js';
export { default as golfDriver } from './icons/golf-driver.js';
export { default as grape, default as grapes } from './icons/grapes.js';
export { default as gridLinesOffset } from './icons/grid-lines-offset.js';
export { default as gridLines } from './icons/grid-lines.js';
export { default as hairdryer } from './icons/hairdryer.js';
export { default as hatBaseball } from './icons/hat-baseball.js';
export { default as hatBeanie } from './icons/hat-beanie.js';
export { default as hatBowler } from './icons/hat-bowler.js';
export { default as chefHat, default as hatChef } from './icons/hat-chef.js';
export { default as hardHat, default as hatHard } from './icons/hat-hard.js';
export { default as hatTop } from './icons/hat-top.js';
export { default as headingCircle } from './icons/heading-circle.js';
export { default as headingSquare } from './icons/heading-square.js';
export { default as hedgehog } from './icons/hedgehog.js';
export { default as helmetDiving } from './icons/helmet-diving.js';
export { default as hexagons3 } from './icons/hexagons-3.js';
export { default as hexagons7 } from './icons/hexagons-7.js';
export { default as highHeel } from './icons/high-heel.js';
export { default as hockeyMask } from './icons/hockey-mask.js';
export { default as hockey } from './icons/hockey.js';
export { default as horseHead } from './icons/horse-head.js';
export { default as hotDog } from './icons/hot-dog.js';
export { default as houseManor } from './icons/house-manor.js';
export { default as houseOff } from './icons/house-off.js';
export { default as houseRoofOff } from './icons/house-roof-off.js';
export { default as houseRoof } from './icons/house-roof.js';
export { default as home, default as house } from './icons/house.js';
export { default as houses } from './icons/houses.js';
export { default as iceHockey } from './icons/ice-hockey.js';
export { default as iceSkate } from './icons/ice-skate.js';
export { default as igloo } from './icons/igloo.js';
export { default as indianRupeeCircle } from './icons/indian-rupee-circle.js';
export { default as indianRupeeSquare } from './icons/indian-rupee-square.js';
export { default as intercom } from './icons/intercom.js';
export { default as ironOff } from './icons/iron-off.js';
export { default as iron } from './icons/iron.js';
export { default as ironingBoard } from './icons/ironing-board.js';
export { default as italicSquare } from './icons/italic-square.js';
export { default as jacketSports } from './icons/jacket-sports.js';
export { default as jacket } from './icons/jacket.js';
export { default as japaneseYenCircle } from './icons/japanese-yen-circle.js';
export { default as japaneseYenSquare } from './icons/japanese-yen-square.js';
export { default as jar } from './icons/jar.js';
export { default as jug } from './icons/jug.js';
export { default as kebab } from './icons/kebab.js';
export { default as kettleElectric } from './icons/kettle-electric.js';
export { default as kettle } from './icons/kettle.js';
export { default as kiwi } from './icons/kiwi.js';
export { default as layoutGridMoveHorizontal } from './icons/layout-grid-move-horizontal.js';
export { default as layoutGridMoveVertical } from './icons/layout-grid-move-vertical.js';
export { default as layoutGridPlus } from './icons/layout-grid-plus.js';
export { default as layoutListMove } from './icons/layout-list-move.js';
export { default as lemon } from './icons/lemon.js';
export { default as lifeJacket } from './icons/life-jacket.js';
export { default as ligatureSquare } from './icons/ligature-square.js';
export { default as lightSwitch } from './icons/light-switch.js';
export { default as lingerie } from './icons/lingerie.js';
export { default as locateSquare } from './icons/locate-square.js';
export { default as luggageCabin } from './icons/luggage-cabin.js';
export { default as lunchBox } from './icons/lunch-box.js';
export { default as mailboxFlag } from './icons/mailbox-flag.js';
export { default as maskSnorkel } from './icons/mask-snorkel.js';
export { default as mealBox } from './icons/meal-box.js';
export { default as mortarPestle } from './icons/mortar-pestle.js';
export { default as motorRacingHelmet } from './icons/motor-racing-helmet.js';
export { default as mugTeabag } from './icons/mug-teabag.js';
export { default as coffee, default as mug } from './icons/mug.js';
export { default as mustache } from './icons/mustache.js';
export { default as olive } from './icons/olive.js';
export { default as onion } from './icons/onion.js';
export { default as owl } from './icons/owl.js';
export { default as pacManGhost } from './icons/pac-man-ghost.js';
export { default as pacMan } from './icons/pac-man.js';
export { default as palmtreeIslandSun } from './icons/palmtree-island-sun.js';
export { default as pancakes } from './icons/pancakes.js';
export { default as peace } from './icons/peace.js';
export { default as peach } from './icons/peach.js';
export { default as pear } from './icons/pear.js';
export { default as penguin } from './icons/penguin.js';
export { default as pepperChilli } from './icons/pepper-chilli.js';
export { default as pie } from './icons/pie.js';
export { default as pigHead } from './icons/pig-head.js';
export { default as pig } from './icons/pig.js';
export { default as pillow } from './icons/pillow.js';
export { default as pinSafetyOpen } from './icons/pin-safety-open.js';
export { default as pinSafety } from './icons/pin-safety.js';
export { default as pineappleRing } from './icons/pineapple-ring.js';
export { default as planet } from './icons/planet.js';
export { default as pond } from './icons/pond.js';
export { default as poundSterlingCircle } from './icons/pound-sterling-circle.js';
export { default as poundSterlingSquare } from './icons/pound-sterling-square.js';
export { default as pram } from './icons/pram.js';
export { default as pretzel } from './icons/pretzel.js';
export { default as pumpkin } from './icons/pumpkin.js';
export { default as razorBlade } from './icons/razor-blade.js';
export { default as razor } from './icons/razor.js';
export { default as reelThread } from './icons/reel-thread.js';
export { default as refrigeratorFreezer } from './icons/refrigerator-freezer.js';
export { default as removeFormattingSquare } from './icons/remove-formatting-square.js';
export { default as rugby } from './icons/rugby.js';
export { default as russianRubleCircle } from './icons/russian-ruble-circle.js';
export { default as russianRubleSquare } from './icons/russian-ruble-square.js';
export { default as sausage } from './icons/sausage.js';
export { default as scarf } from './icons/scarf.js';
export { default as scissorsHairComb } from './icons/scissors-hair-comb.js';
export { default as shark } from './icons/shark.js';
export { default as shaveFace } from './icons/shave-face.js';
export { default as shirtFoldedButtons } from './icons/shirt-folded-buttons.js';
export { default as shirtLongSleeve } from './icons/shirt-long-sleeve.js';
export { default as shirtTRuler } from './icons/shirt-t-ruler.js';
export { default as shirtTVNeck } from './icons/shirt-t-v-neck.js';
export { default as shirt, default as shirtT } from './icons/shirt-t.js';
export { default as shortsBoxer } from './icons/shorts-boxer.js';
export { default as shorts } from './icons/shorts.js';
export { default as shovelDig } from './icons/shovel-dig.js';
export { default as shower } from './icons/shower.js';
export { default as skirt } from './icons/skirt.js';
export { default as skis } from './icons/skis.js';
export { default as slotCardCredit } from './icons/slot-card-credit.js';
export { default as slotCard } from './icons/slot-card.js';
export { default as slotDisc } from './icons/slot-disc.js';
export { default as sneaker } from './icons/sneaker.js';
export { default as snowboard } from './icons/snowboard.js';
export { default as snowman } from './icons/snowman.js';
export { default as soapBar } from './icons/soap-bar.js';
export { default as soccerBall } from './icons/soccer-ball.js';
export { default as soccerPitch } from './icons/soccer-pitch.js';
export { default as socketEu } from './icons/socket-eu.js';
export { default as socketUk } from './icons/socket-uk.js';
export { default as socketUsa } from './icons/socket-usa.js';
export { default as socks } from './icons/socks.js';
export { default as spiderWeb } from './icons/spider-web.js';
export { default as spider } from './icons/spider.js';
export { default as stairsArch } from './icons/stairs-arch.js';
export { default as stairsArrowDownLeft } from './icons/stairs-arrow-down-left.js';
export { default as stairsArrowUpRight } from './icons/stairs-arrow-up-right.js';
export { default as stairs } from './icons/stairs.js';
export { default as starNorth } from './icons/star-north.js';
export { default as steeringWheel } from './icons/steering-wheel.js';
export { default as strawberry } from './icons/strawberry.js';
export { default as strikethroughSquare } from './icons/strikethrough-square.js';
export { default as stroller } from './icons/stroller.js';
export { default as sunloungerParasolSunPalmtree } from './icons/sunlounger-parasol-sun-palmtree.js';
export { default as sunloungerParasolSun } from './icons/sunlounger-parasol-sun.js';
export { default as sunloungerParasolTable } from './icons/sunlounger-parasol-table.js';
export { default as surfboard } from './icons/surfboard.js';
export { default as sushi2 } from './icons/sushi-2.js';
export { default as sushi3 } from './icons/sushi-3.js';
export { default as sushiChopsticks } from './icons/sushi-chopsticks.js';
export { default as sushi } from './icons/sushi.js';
export { default as sweater } from './icons/sweater.js';
export { default as swissFrancCircle } from './icons/swiss-franc-circle.js';
export { default as swissFrancSquare } from './icons/swiss-franc-square.js';
export { default as tabArrowDown } from './icons/tab-arrow-down.js';
export { default as tabArrowUpRight } from './icons/tab-arrow-up-right.js';
export { default as tabCheck, default as vote } from './icons/tab-check.js';
export { default as tabDot } from './icons/tab-dot.js';
export { default as tabPlus } from './icons/tab-plus.js';
export { default as tabSlash } from './icons/tab-slash.js';
export { default as tabText } from './icons/tab-text.js';
export { default as tabX } from './icons/tab-x.js';
export { default as tab } from './icons/tab.js';
export { default as targetArrow } from './icons/target-arrow.js';
export { default as tennisBall } from './icons/tennis-ball.js';
export { default as tennisRacket } from './icons/tennis-racket.js';
export { default as textSquare } from './icons/text-square.js';
export { default as tieBowRibbon } from './icons/tie-bow-ribbon.js';
export { default as tieBow } from './icons/tie-bow.js';
export { default as tie } from './icons/tie.js';
export { default as tire } from './icons/tire.js';
export { default as toast } from './icons/toast.js';
export { default as toaster } from './icons/toaster.js';
export { default as toiletRoll } from './icons/toilet-roll.js';
export { default as toolbox2 } from './icons/toolbox-2.js';
export { default as toolbox } from './icons/toolbox.js';
export { default as topCrop } from './icons/top-crop.js';
export { default as towelFolded } from './icons/towel-folded.js';
export { default as towelRack } from './icons/towel-rack.js';
export { default as treesForest } from './icons/trees-forest.js';
export { default as triangleStripes } from './icons/triangle-stripes.js';
export { default as trousers } from './icons/trousers.js';
export { default as tuxedo } from './icons/tuxedo.js';
export { default as typeSquare } from './icons/type-square.js';
export { default as ufo } from './icons/ufo.js';
export { default as underlineSquare } from './icons/underline-square.js';
export { default as unicornHead } from './icons/unicorn-head.js';
export { default as venn } from './icons/venn.js';
export { default as vest } from './icons/vest.js';
export { default as volleyball } from './icons/volleyball.js';
export { default as waffle } from './icons/waffle.js';
export { default as wardrobe } from './icons/wardrobe.js';
export { default as watchActivity } from './icons/watch-activity.js';
export { default as watchAlarm } from './icons/watch-alarm.js';
export { default as watchBars } from './icons/watch-bars.js';
export { default as watchCharging } from './icons/watch-charging.js';
export { default as watchCheck } from './icons/watch-check.js';
export { default as watchLoader } from './icons/watch-loader.js';
export { default as watchMusic } from './icons/watch-music.js';
export { default as watchSquareAlarm } from './icons/watch-square-alarm.js';
export { default as watchSquare } from './icons/watch-square.js';
export { default as watchText } from './icons/watch-text.js';
export { default as watermelon } from './icons/watermelon.js';
export { default as waveCircle } from './icons/wave-circle.js';
export { default as wavesBirds } from './icons/waves-birds.js';
export { default as wavesSharkFin } from './icons/waves-shark-fin.js';
export { default as whaleNarwhal } from './icons/whale-narwhal.js';
export { default as whale } from './icons/whale.js';
export { default as wheel } from './icons/wheel.js';
export { default as whiskForkKnife } from './icons/whisk-fork-knife.js';
export { default as whisk } from './icons/whisk.js';
export { default as whisks } from './icons/whisks.js';
export { default as windmill } from './icons/windmill.js';
export { default as wineGlassBottle } from './icons/wine-glass-bottle.js';
export { default as yarnBall } from './icons/yarn-ball.js';
export { default as yinYang } from './icons/yin-yang.js';
